sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDatabase.java

```diff
@@ -458,7 +497,7 @@ private CosmosUserResponse convertUserResponse(CosmosAsyncUserResponse response)
     /**
      * Read provisioned throughput integer.
      *
-     * @return the integer. null response indicates database doesn't have any provisioned RUs
+     * @return the integer. null response indicates database doesn't have any provisioned RUs.
      */
     public Integer readProvisionedThroughput() {
```

Comment:
Same here - hide / remove this API - https://github.com/Azure/azure-sdk-for-java/issues/11191

Reply:
Fixed.